app/models/invenio_rdm_record_converter.rb

```diff
@@ -200,4 +202,8 @@ def additional_descriptions(descriptions)
     return nil if additional_descriptions_size < 0
     descriptions.last(additional_descriptions_size).map{ |add_desc| {"description": add_desc, "type": "other", "lang": "eng"} }
   end
+
+  def funding(file_id)
+    @@funding_data[file_id] || {}
```

Comment:
If I remember right we're no longer returning nil so I figured a blank hash would do.